db/migrate/20200221231419_index_active_exercise_calculations_on_student_uuid_and_ecosystem_uuid.rb

```diff
@@ -0,0 +1,7 @@
+class IndexActiveExerciseCalculationsOnStudentUuidAndEcosystemUuid < ActiveRecord::Migration[5.2]
+  def change
+    add_index :exercise_calculations, [ :student_uuid, :ecosystem_uuid ],
+              where: '"superseded_at" IS NULL',
+              name: 'index_active_ex_calc_on_student_uuid_and_ecosystem_uuid'
+  end
+end
```

Comment:
how large is `exercise_calculations` ?  should we do this in background?

Reply:
About half a million records, we decided to test how long it takes on staging.